config/initializers/filter_parameter_logging.rb

```diff
@@ -2,5 +2,5 @@
 
 # Configure sensitive parameters which will be filtered from the log file.
 Rails.application.config.filter_parameters += %i[
-  passw secret token _key crypt salt certificate otp ssn
```

Comment:
:joy:  Don't want to ask how much debugging frustration finding this config took